test/pythonml/test_model_flavors.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python
+import importlib
+import os
+import shutil
+import unittest
+from parameterized import parameterized
+from fixtures.unittest_baseclass import BaseUnitTest
+
+
+
+class TestModelFlavors(BaseUnitTest):
+    """
+    This class performs unit tests for the model flavors
+    """
+
+    def check_pass_conditions_training(self, category):
+        import settings
+        importlib.reload(settings)
+        with settings.context as context:
+            if category == 'regression':
+                rmse = context.load('RMSE')
+                assert(rmse <= 2 * 10)
+            elif category == 'classification':
+                confusion_matrix = context.load('confusion_matrix')
+                accuracy = confusion_matrix.diagonal() / confusion_matrix.sum(axis=0)
+                assert (accuracy.all() >= 0.6)
+
+    def check_pass_conditions_predicting(self):
+        assert (os.path.isfile('predictions.csv'))
+
+    def run_flavor_test(self, category, flavor, do_predict):
```

Comment:
Generally, our convention is to have booleans prefixed by `is` when possible, such as `is_predicting` or something here. Helps with readability to stay consistent with that